lms/djangoapps/certificates/views/support.py

```diff
@@ -208,6 +208,17 @@ def regenerate_certificate_for_user(request):
         )
         return HttpResponseServerError(_("An unexpected error occurred while regenerating certificates."))
 
+    try:
+        # Fetch CertificateInvalidation object
+        certificate_invalidation = CertificateInvalidation.objects.get(
+            generated_certificate=certificate,
+            active=True
+        )
+        # Deactivate certificate invalidation if it was fetched successfully.
+        certificate_invalidation.deactivate()
+    except:  # pylint: disable=bare-except
+        pass
```

Comment:
@schenedx I was just checking certificate_invalidation validation exists or not.  Now I will catch the DoesNotExist here.